source/sonata_simulation.rst

```diff
@@ -476,6 +477,8 @@ Collection of dictionaries to adjust the synaptic strength or other properties o
    modoverride                    text       Optional     Changes the synapse helper files used to instantiate the synapses in this connection. A synapse helper initializes the synapse object and the parameters of the synapse model. By default, AMPANMDAHelper.hoc / GABAABHelper.hoc are used for excitatory / inhibitory synapses. The value of this field determines the prefix of the helper file to use e.g. "Newfun" would lead to NewfunHelper.hoc being used. Exceptionally, passing "GluSynapse" will lead to GluSynapse.hoc being used. That helper will use the additional parameters of the plastic synapse model read from the SONATA edges file using Neurodamus. This is required when using the GluSynapse.mod model and will fail for other models, or if the parameters are not present in the edges file.
    synapse_delay_override         float      Optional     Value to override the synaptic delay time originally set in the edge file, and to be given to netcon object. Given in ms.
    delay                          float      Optional     Adjustments from weight of this connection_override are applied after specified delay has elapsed in ms. Note that only weight modifications are applied so all other fields (spont_minis, synapse_configure, modoverride, synapse_delay_override) are ignored.
+   neuromod_dtc                   float      Optional     Only applicable to NeuroModulation projections. It overrides the "neuromod_dtc" values between the selected source and target neurons, representing the decay time constant of the neuromodulator concentration at the target synapse.
```

Comment:
would it make sense to use a longer name for this: `neuromodulation_dtc` since `mod` is a common word, but that refers to `mod` files and not neuro modulation? (same w/ neuromod_strength)

Reply:
OK. I've made the change.